app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -1,35 +1,23 @@
 import { useContext } from 'react';
 import { PageHeading } from '@18f/identity-components';
-import {
-  FormStepError,
-  FormStepsButton,
-  OnErrorCallback,
-  RegisterFieldCallback,
-} from '@18f/identity-form-steps';
+import { FormStepsButton } from '@18f/identity-form-steps';
 import { Cancel } from '@18f/identity-verify-flow';
 import { useI18n } from '@18f/identity-react-i18n';
+import type { FormStepComponentProps } from '@18f/identity-form-steps';
 import UnknownError from './unknown-error';
 import TipList from './tip-list';
 import DocumentSideAcuantCapture from './document-side-acuant-capture';
 import DocumentCaptureNotReady from './document-capture-not-ready';
 import { FeatureFlagContext } from '../context';
 import DocumentCaptureAbandon from './document-capture-abandon';
 import { DocumentCaptureSubheaderOne, SelfieStepWithHeader } from './documents-step';
+import type { ReviewIssuesStepValue } from './review-issues-step';
 
-interface DocumentCaptureReviewIssuesProps {
+interface DocumentCaptureReviewIssuesProps
+  extends Omit<FormStepComponentProps<ReviewIssuesStepValue>, 'toPreviousStep'> {
```

Comment:
Why do we omit `toPreviousStep`? I'd expect it would be expected to be passed if `FormSteps` is rendering it, even if it's not used by the component here?

Reply:
Didn't see this until after merge, fixed this with the [PR here](https://github.com/18F/identity-idp/pull/9852).